docs/openshift-installation.md

```diff
@@ -0,0 +1,18 @@
+# Openshift Installation
+
+This installation method is the recommended way for Openshift users. **Note**: Openshift version must be 4.2 or higher.
+
+The NGINX Ingress Operator is a [RedHat certified Operator](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-operator-certification).
+
+1. In the Openshift dashboard, click in `Operators` > `Operator Hub` in the left menu and use the search box to type `nginx ingress`:
+![alt text](./images/openshift1.png "Operators")
+1. Click in the NGINX Ingress Operator and click Install:
```

Comment:
should we highlight the names of menus, buttons, etc in all sections, note only in the previous one?  Perhaps in **bold** or *italic*?